aptos-move/framework/aptos-framework/sources/account.move

```diff
@@ -1046,21 +1156,35 @@ module aptos_framework::account {
     public entry fun test_empty_public_key(alice: signer) acquires Account, OriginatingAddress {
         create_account(signer::address_of(&alice));
         let pk = vector::empty<u8>();
-        let sig = x"00000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000";
-        rotate_authentication_key(&alice, ED25519_SCHEME, pk, ED25519_SCHEME, pk, sig, sig);
+        let sig =
+            x"00000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000";
+        rotate_authentication_key(
+            &alice, ED25519_SCHEME, pk, ED25519_SCHEME, pk, sig, sig
+        );
     }
 
     #[test(alice = @0xa11ce)]
     #[expected_failure(abort_code = 262151, location = Self)]
     public entry fun test_empty_signature(alice: signer) acquires Account, OriginatingAddress {
         create_account(signer::address_of(&alice));
         let test_signature = vector::empty<u8>();
-        let pk = x"0000000000000000000000000000000000000000000000000000000000000000";
-        rotate_authentication_key(&alice, ED25519_SCHEME, pk, ED25519_SCHEME, pk, test_signature, test_signature);
+        let pk =
+            x"0000000000000000000000000000000000000000000000000000000000000000";
+        rotate_authentication_key(
+            &alice,
+            ED25519_SCHEME,
+            pk,
+            ED25519_SCHEME,
+            pk,
+            test_signature,
+            test_signature,
+        );
```

Comment:
It's interesting that these were treated differently

Reply:
It will consider respecting the input in input.move; if the function call itself is short, it will adopt a single-line mode. You can refer to this function in the movefmt source code: call_fmt::get_call_component_split_mode(). In fact, many cases that seem strange are due to complex judgments, as it is necessary to be compatible with most scenarios.